### ISSUE TITLE
Opacity?

### ISSUE THREAD

**Issue by jeetdholakia** [silvaren/easyrs#7]
Is there a way to set opacity on either LUT or 3D LUT? The Android docs do not have an alpha channel in 3D LUT but they do for LUT...

**Comment by silvaren** [comment]
Hi @jeetdholakia ! Yes they should work with alpha channels, check https://github.com/silvaren/easyrs/blob/master/easyrs/src/main/java/io/github/silvaren/easyrs/tools/params/SampleParams.java for an example on how to build the LUTs to use with the library.

**Comment by jeetdholakia** [comment]
Yes, I tried using the opacity parameter but the more opacity I used, the whiter it made the image. What I was hoping for is a result similar to choosing filter strength in photo editing apps, but I guess the Android SDK does not work like that. Any guesses?
